### DISCUSSION TITLE
Hope for update

### DISCUSSION THREAD

**Discussion by MichaelLiew1990** [shun126/UE5-DungeonGeneratorDemo#25] [Q&A] [answered]
These features we need, hope you update if you have time, thanks!
1. DungeonGenerateActor cannot move far away from [0,0,0], it always spawn in [0,0,0]
2. We need Multi-DungeonGenerateActor feature and can place these actors anywhere at runtime manually
3. Multi-StartLevel, Multi-GoalLevel
4. Door, Ramp, Pickups, can show on the minimap

Your plugin is great!

**Comment by shun126** [comment]
Displaying icons on the mini-map in #4 can be made in the current version of the blueprint, but a convenient function should be added to the plugin.

At the same time, add a widget class for icons

**Reply by shun126** [reply, marked as answer]
> 4. Door, Ramp, Pickups, can show on the minimap

You can display icons on the minimap using DungeonIconWidget. The Demo project displays icons, so it should be helpful as a reference.

**Comment by shun126** [comment]
The original concept was intended to be a puzzle with MissionGraph, so Multi-StartLevel, Multi-GoalLevel is the most difficult to realize.

**Reply by shun126** [reply]
Multi-StartLevel and Multi-GoalLevel are difficult to support from the current implementation.
Multi-GoalLevel can be achieved to some extent by placing a sublevel where the goal object is placed.
However, if the goal object fails the lottery, it will not be placed, so it may not meet your request.

Multi-StartLevel, Multi-GoalLevel は現在の実装から対応するのが難しいです。
Multi-GoalLevel は、ゴールオブジェクトを置いたサブレベルが配置する事で、ある程度は実現できます。
抽選に落ちると配置されないので要望にそえないかもしれません。

**Comment by shun126** [comment]
> 1. DungeonGenerateActor cannot move far away from [0,0,0], it always spawn in [0,0,0]

Regarding placement at locations other than the origin, this has been addressed in version 1.6.3.

> 3. Multi-StartLevel, Multi-GoalLevel

It seems possible to make a room with only one corridor (called "Hanare" in Japanese) a start room or a goal room.
The problem is how to determine the number of start and goal points. I can't imagine it.
